.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
- [ ] **Required:** Kindly replace this linter files with the appropriate one [here.](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml)

Reply:
Hi, @brytebee 

This has been well received and acted upon, however, the project requirements only mentioned HTML and CSS linter. The file provided [here](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) includes even JavaScript Linters. 

**Please advise on this.**

Reply:
Hello @Felix45 Yes in this project you just need linters for html and css only.